domains/DataCollection/index.js

```diff
@@ -196,6 +196,7 @@ const DataCollection = ({ navigation }) => {
                     surveyee={surveyee}
                     setSurveyee={setSurveyee}
                     customForm={customForm}
+                    setCustomForm={setCustomForm}
```

Comment:
what's going on here?

Reply:
seems like you're passing down `setCustomForm()` but not using it? haha I might be missing something

Reply:
nah youre right, must have missed that one when i was cleaning up.